Gems/EMotionFX/Code/Source/Editor/Plugins/ColliderWidgets/JointPropertyWidget.cpp

```diff
@@ -342,6 +347,25 @@ namespace EMotionFX
             }
         }
 
+        // paste a copied collider
```

Comment:
Nit: use Upper case comment
// Paste a copied collider

Same for other comments.